pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - 'nodes/**'
```

Comment:
these aren't separate packages, and this repo isn't a mono-repo. So, this files does not make sense.